applications/matter_bridge/doc/matter_bridge_description.rst

```diff
@@ -137,21 +142,52 @@ For details, see the `Testing`_ section.
 User interface
 **************
 
-Button 1:
-   .. include:: /includes/matter_sample_button.txt
+.. tabs::
+   .. group-tab:: nRF70 and nRF53 DKs
```

Comment:
Just a nit here - in other samples we were usually numbering from the lowest to the highest number
```suggestion
   .. group-tab:: nRF53 and nRF70 DKs
```